Configuration/TypoScript/setup.txt

```diff
@@ -17,7 +17,7 @@ page.includeJSFooter.mindshapecookiehint = EXT:mindshape_cookie_hint/Resources/P
 page.9877 = USER
 page.9877 {
   userFunc = TYPO3\CMS\Extbase\Core\Bootstrap->run
-  pluginName = Main
+  pluginName = CookieHint
```

Comment:
Why did you change the plugin name or why was this necessary?

Reply:
In the definitions for configurePlugin and registerPlugin have been some changes required and complaints in InstallTool never have been all reasonable for me. I changed things so long till it worked. So i general it could work if you replace "CookieHint" everywhere with "Main", but "CookieHint" is more descriptive anyway in my opinion.